### PR TITLE
Add additional session tracking scenario

### PR DESCRIPTION
**Pull request by fractalwrench**
Adds a case where a user sets auto capture after initialisation

### CODE REVIEW

File: mazerunner/src/main/java/com/bugsnag/android/mazerunner/scenarios/SessionSetAutoCaptureScenario.kt [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ * Sets automatic capture of sessions in Bugsnag and flushes 1 session
+ */
+internal class SessionSetAutoCaptureScenario(config: Configuration,
+                                             context: Context) : Scenario(config, context) {
+
+    override fun run() {
+        super.run()
+        Bugsnag.setAutoCaptureSessions(true)
+        context.startActivity(Intent(context, SecondActivity::class.java))
+        flushAllSessions()
```
[the remark this answers is not in the document]
Comment (fractalwrench) [reply mazerunner/src/main/java/com/bugsnag/android/mazerunner/scenarios/SessionSetAutoCaptureScenario.kt]: 60 seconds by default, I've updated the wait time.

[the remark this answers is not in the document]
Comment (kattrali) [reply mazerunner/src/main/java/com/bugsnag/android/mazerunner/scenarios/SessionSetAutoCaptureScenario.kt]: Why not also remove `flushAllSessions()`?

[the remark this answers is not in the document]
Comment (fractalwrench) [reply mazerunner/src/main/java/com/bugsnag/android/mazerunner/scenarios/SessionSetAutoCaptureScenario.kt]: Good question, I've removed that call.

**Comment by fractalwrench** [comment]
@kattrali I believe I've addressed all the review comments on this one.

**Pull request merged**